recipes/test.rb

```diff
@@ -28,7 +28,19 @@
 include_recipe 'jenkins::master'
 
 jenkins_plugins_workflow 'workflow-job' do
-  script 'hello-world.groovy'
+  script 'workflow/hello-world.groovy'
 end
 
-jenkins_plugins_maven 'M3'
+jenkins_plugins_maven 'M3' do
+  install true
+end
+
+jenkins_plugins_ssh 'sam' do
+  host 'bitbucket.org'
+  hostname 'altssh.bitbucket.org'
+  port 443
+  type 'rsa'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.